umm-spec-lib/test/cmr/umm_spec/test/generate_and_parse.clj

```diff
@@ -170,6 +170,24 @@
       (is (= expected actual)
           (str "Unable to roundtrip with format " metadata-format)))))
 
+(deftest roundtrip-generated-collection-records-with-seed
+  (checking-with-seed "collection round tripping seed" 100 1496683985472
```

Comment:
Could you write a brief doc to describe how this test is intended to be used when the intermittent failure happens in the old roundtrip-generated-collection-records, which you want to preserve to reveal potential issues in the future?